src/docker/server/Dockerfile

```
@@ -1,12 +1,14 @@
 # Copyright 2023 Flower Labs GmbH. All Rights Reserved.
 
 ARG BASE_REPOSITORY=flwr/base
-ARG BASE_IMAGE_TAG=py3.11-ubuntu22.04
-FROM $BASE_REPOSITORY:$BASE_IMAGE_TAG as server
+ARG PYTHON_VERSION=3.10
+ARG ALPINE_VERSION=alpine3.19
+FROM ${BASE_REPOSITORY}:py${PYTHON_VERSION}-${ALPINE_VERSION} as server
 
-WORKDIR /app
 ARG FLWR_VERSION
-RUN python -m pip install -U --no-cache-dir flwr[rest]==${FLWR_VERSION}
+RUN pip install -U --no-cache-dir \
+    flwr[rest]==${FLWR_VERSION:-1.7.0}
+
 ENTRYPOINT ["python", "-c", "from flwr.server import run_superlink; run_superlink()"]
```

We can now update this to 
```suggestion
ENTRYPOINT ["flower-server"]
```
Note: it will not work if create the non-root user before we install flwr.